check_tag.py

```diff
@@ -0,0 +1,59 @@
+"""
+This file contains functions that checks if tag in git and in
+setup.py are same. If they are not an exception is raised.
+"""
+
+import re
+import subprocess
+
+
+def check_tag():
+    """
+    This function checks if tag in git and in setup.py are same and
+    returns a boolean
+    """
+    setuppy_version = get_tag_in_setuppy("setup.py")
+    tag_version = get_latest_tag()
+    print("The version number in setup.py is : ", setuppy_version)
+    print("The version number in git tag is : ", tag_version)
+    return setuppy_version == tag_version
+
+def get_latest_tag():
+    """
+    This function gets the tag from git and formats it as a string
+    for later use.
+    """
+    proc = subprocess.Popen(["git", "describe", "--tag"],\
+                    stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout, _ = proc.communicate()
+    version = stdout.strip().decode('utf-8')
+    pattern = r"v(\d+).(\d+).(\d+)(.*?)"
+    match = re.match(pattern, version)
+    if match:
+        major = match.group(1)
+        minor = match.group(2)
+        build = match.group(3)
+        return "v"+".".join((major, minor, build))
```

Comment:
For this and the next function, why do you need to split into minor-major-build? Don't you want to check check if the tag names are the same, regardless of their format ?

Reply:
I initially wanted to get the three numbers and compare them. Finally settled down to comparing the string. I will change it.